src/common/config.go

```diff
@@ -218,6 +218,12 @@ func checkConf() error {
 		return fmt.Errorf("Unknown Sandbox type '%s'", Conf.Sandbox)
 	}
 
+	// check proc number limit. if proc num == 1, then any fork will fail.
+	// TODO: (Discussion) what is the proc num we shall set as a hard lower bound?
```

Comment:
@tylerharter I think we should add a proc limit here to safe-check the config. Apperantly, `proc<10` means the system is basically useless (it will crash after 3~4 plain requests). But whether we shall set this here is worth discussion.